cmd/operator/main.go

```diff
@@ -50,7 +50,7 @@ func main() {
 		"namespace", namespace,
 		"metrics-bind-address", metricsAddr)
 
-	op, err := operator.New(metricsAddr)
+	op, err := operator.New(metricsAddr, ":9090")
```

Comment:
Do we want it as a config option as well? And what about the default value? Is `9090` OK?

Reply:
An argument would be handy. I can't think of a reason for changing the default but at least we'll be safe. Maybe pick 8081 (instead of 9090) to follow 8080.